### ISSUE TITLE
Getting error message when launching app from command prompt with "disable-extension" argument.

### ISSUE THREAD

**Issue by v-pavanp** [Microsoft/vscode#4536]
- VSCode Version: 0.10.12-alpha
- OS Version: Windows10

Steps to Reproduce:

1.  Launch an app from command prompt without any command line argument.
>code-alpha
2. Open any folder which contains C# code.
3. Verify installed extensions for C#, PS, GO etc. should work fine.
4. Verify there should not be any error message. 
5. Close app and re-launch it from command prompt with disable-extension argument.
>code-alpha --disable-extension
5. Verify app behavior and configuration changes.

Actual:
1. Getting error message Like: "Cannot convert undefined or null to object"
 
![image](https://cloud.githubusercontent.com/assets/17735796/13939789/40f63f90-ef95-11e5-93a7-a2b75089d8f6.png)

2. There is no changes in app behavior or configuration. 
a. Default setting is not changing at all.
b. Linter is working as it was working before re-launch.
c. Debugger is working as it was working before re-launch.
d. Intellisense is working as it was working before re-launch.
 
Expected: There should not be any error message.

Clarification: What are the changes user should expect when launching app with --disable-extension argument.

**Comment by bpasero** [comment]
Possibly a duplicate of https://github.com/Microsoft/vscode/issues/4526 ?

@v-pavanp having a hard time to reproduce, are you able to "break on error" in the developer console and try again? can you send me the workspace where this happens?

**Comment by v-pavanp** [comment]
This is only with my windows10 machine. I tried to verify same on windows8 but it is working as expected.
I tried by completely uninstalling app from control panel on windows10 machine but still getting the same error message. The main difference In my windows10 and winfow8 machine is number of installed extensions.
WIndows10: c# ,Go , Powershell, Debugger for chrome, ESlint, HTML Hint, Spell and Grammar check, Markdown Theme kit, Mono debug, JS-CSS-HTML formatter, Ruby,Ruby-linter
Win8: C#
